tutorials/text/distributed-data-classification/content-type-classification.ipynb

```diff
@@ -8,16 +8,20 @@
     "\n",
     "This notebook demonstrates the use of NeMo Curator's `ContentTypeClassifier`. The [content type classifier](https://huggingface.co/nvidia/content-type-classifier-deberta) is used to categorize documents into one of 11 distinct speech types based on their content. It helps with data annotation, which is useful in data blending for foundation model training. Please refer to the NemoCurator Content Type Classifier DeBERTa Hugging Face page for more information about the content type classifier, including its output labels, here: https://huggingface.co/nvidia/content-type-classifier-deberta.\n",
```

Comment:
I think the ask is to also add another type of text and also explain what labels we have here. 

Lets add one like this example:
```
text_samples = ["Hi, great video! I am now a subscriber."]
```

And also add what labels are possible here